src/components/Form/CalendarInput/CalendarInput.js

```diff
@@ -197,23 +199,47 @@ CalendarInput.defaultProps = {
 };
 
 CalendarInput.propTypes = {
+    /** If true ,there is a "x" mark inside the component,
+     *  if "x" mark is clicked the element inside the component will be cleared,
+     *  if false there is no "x" mark in the component.*/
     clearable: PropTypes.bool,
-    dateFormat: PropTypes.oneOfType([
-        PropTypes.string,
-        PropTypes.bool
-    ]),
+
+    /** If True, users won't be able to interact with the component*/
     disabled: PropTypes.bool,
+
     errors: PropTypes.objectOf(
         PropTypes.arrayOf(PropTypes.string)
     ),
+
+    /** Layout that arranges views of the component*/
     gridClass: PropTypes.string,
+
+    /** Specifies the icon class to be displayed within the component*/
     iconClass: PropTypes.string,
+
+    /** Specify a callback to determine the dates that can be selected.
+     * The function receives (currentDate, selectedDate) and shall return
+     * a true or false whether the date is valid or not.*/
     isSelectableDate: PropTypes.func,
+
+    /** Specifies the text to use as the label*/
     label: PropTypes.node,
+
+    /** Specifies the name of the component. It is used to distinguish elements when
+     * a single form change handler is used*/
     name: PropTypes.string.isRequired,
+
+    /** Callback fired when component value changes. Accepts a function with two parameters,
+     *  namely field and value*/
     onChange: PropTypes.func,
+
+    /** Set to false to use date picker only*/
```

Comment:
Description interchanged with timepicker description. (i.e., we need to set the `datepicker` to `false` to show time picker only, and `timepicker` to `false` to show date picker only.